### PR TITLE
fix: add shadow in switch of TOS

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
add shadow in switch of TOS
## List of changes proposed in this pull request
- add custom style for switch

## How to test
see switch with shadow in page Tos

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx [hunk 1]
```diff
@@ -91,6 +91,11 @@ const TermsOfService = ({ tosConsent, privacyConsent }: TermsOfServiceProps) =>
             <Box display="flex" alignItems="center">
               <Switch
                 value={accepted}
+                sx={{
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx:94]: Remove this code. Now it is useless

File: packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx [hunk 2]
```diff
@@ -91,6 +91,11 @@ const TermsOfService = ({ tosConsent, privacyConsent }: TermsOfServiceProps) =>
             <Box display="flex" alignItems="center">
               <Switch
                 value={accepted}
+                sx={{
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx:94]: Same as before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]
Also PA must be updated with new version of mui-italia

File: packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx (hunk 1 above)
Comment (leleOFA) [reply packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx]: done

File: packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx]: done

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**